fix: Keyboard navigation in area chart to allow highlighting all poin…

jperals opened the pull request:
…ts at a given X coordinate

### Description

<!-- Include a summary of the changes and the related issue. -->

Similar to [PR 538](https://github.com/cloudscape-design/components/pull/538) with LineChart, this PR adds a focused state to AreaChart where all points from all series at a given X coordinate can be highlighted with the keyboard:

<img width="731" alt="Screenshot 2022-12-02 at 15 18 17" src="https://user-images.githubusercontent.com/1257272/205313230-85fb3c02-65cb-4102-84e8-9ffbecf1ca12.png">

The specific series can then be highlighted by pressing the Up and Down keys. At the end of the array we cycle back to highlighting all points.

Related ticket: AWSUI-19257

### How has this been tested?

<!-- How did you test to verify your changes? -->

<!-- How can reviewers test these changes efficiently? -->

- Added integration tests and adapted some as necessary to the new interaction
- Manually tested

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

rubencarvalho commented on src/area-chart/__integ__/area-chart.test.ts:
```diff
@@ -259,13 +297,32 @@ describe('Focus delegation', () => {
+      await page.keys(['ArrowRight', 'ArrowRight', 'Enter']);
+
+      await expect(page.getHighlightedSeriesLabel()).resolves.toBe(null);
+      await expect(page.getPopoverTitle()).resolves.toBe('3s');
```
After two `ArrowRight`, shouldn't this be `2s`?  Or am I missing something?

jperals replied:
After calling `await page.focusPlot()`, the focus is already on the `1s` because this function already types `ArrowRight` once. So after going right two more times we are at `3s`.

rubencarvalho reviewed (approved):
LGTM 👍  - just left a comment on one of the tests.

jperals commented:
As per discussion with @avinashbot, removed the aria-label that I had added on the VerticalMarker, same as  for the Line Chart, see https://github.com/cloudscape-design/components/pull/538#discussion_r1041176326
